docs/ui/components/Sidebar/SidebarCollapsible.tsx

```diff
@@ -82,14 +60,68 @@ export const SidebarCollapsible = ({ info, children }: SidebarCollapsibleProps)
 
   return (
     <>
-      <a css={STYLES_TITLE} onClick={toggleIsOpen}>
-        <ChevronDownIcon
-          size={iconSize.small}
-          css={[STYLES_CHEVRON_ICON, !isOpen && STYLES_CLOSED_CHEVRON_ICON]}
-        />
-        {info.name}
+      <a css={titleStyle} onClick={toggleIsOpen}>
+        <div css={chevronContainerStyle}>
+          <ChevronDownIcon
+            size={iconSize.small}
+            css={[chevronStyle, !isOpen && chevronClosedStyle]}
+          />
+        </div>
+        <CALLOUT weight="semiBold">{info.name}</CALLOUT>
       </a>
       {isOpen && <div>{children}</div>}
     </>
   );
-};
+}
+
+const chevronStyle = css({
+  transition: '100ms ease transform',
+});
+
+const titleStyle = css({
+  display: 'flex',
+  alignItems: 'center',
+  gap: spacing[2],
+  position: 'relative',
+  marginBottom: spacing[2],
+  marginLeft: -spacing[2],
+  marginRight: -spacing[2],
```

Comment:
```suggestion
  marginLeft: -spacing[2],
  marginRight: -spacing[2],
```

I would like to avoid negative margin until it is no other option to style the element.